kotlin-analysis-api/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ description = "Kotlin Symbol Processing implementation using Kotlin Analysis API
 
 val intellijVersion: String by project
 val junitVersion: String by project
-val analysisAPIVersion = "1.7.0-dev-1984"
+val analysisAPIVersion = "1.7.0-dev-2437"
```

Comment:
May I know why we need a different variable for this? Looks like it is the same as `kotlinBaseVersion`.

Reply:
Also could you move it to `gradle.properties` if the variable needs to exist alongside with `kotlinBaseVersion`?

Reply:
the release frequency of analysis API and bootstrap compiler were different. I will remove this in a separate PR later.

Reply:
Sounds good. BTW, should the compiler and stdlib version stick with `kotlinBaseVersion` instead of using `analysisAPIVersion` below?

Reply:
maybe just make everything depending on `kotlinBaseVersion`?